src/module/item/condition/index.ts

```diff
@@ -51,11 +50,10 @@ class ConditionPF2e extends AbstractEffectPF2e {
     }
 
     /** Set a self roll option for this condition */
-    override prepareActorData(this: Embedded<ConditionPF2e>): void {
-        const slug = this.slug ?? sluggify(this.name);
-        this.actor.rollOptions.all[`self:condition:${slug}`] = true;
+    override prepareActorData(): void {
+        super.prepareActorData();
         if (this.slug === "flat-footed") {
-            this.actor.rollOptions.all["self:flatFooted"] = true;
+            this.actor!.rollOptions.all["self:flatFooted"] = true;
```

Comment:
Let's get rid of this: it's a backward compatibility holdover for data that has since been migrated.

Reply:
Fair, good to know.